java/code/src/com/redhat/rhn/frontend/xmlrpc/channel/software/ChannelSoftwareHandler.java

```diff
@@ -1029,7 +1029,7 @@ public int setUserSubscribable(User loggedInUser, String channelLabel,
             throw new PermissionCheckFailureException();
         }
 
-        if (value) {
+        if (Boolean.TRUE.equals(value)) {
```

Comment:
Can't we convert `value` to `bool` instead?

Reply:
same as above